dev/create-release/announce.tmpl

```diff
@@ -0,0 +1,16 @@
+We are happy to announce the availability of Spark {version}!
+
+Spark {version} is a maintenance release containing stability fixes. This
+release is based on the {branch} maintenance branch of Spark. We strongly
+recommend all {short_version} users to upgrade to this stable release.
+
+To download Spark {version}, head over to the download page:
+https://spark.apache.org/downloads.html
```

Comment:
Hi, @srowen . Could you review this template? This PR will guarantee that new release managers will use HTTPS.